src/mmds/src/token.rs

```diff
@@ -368,17 +372,6 @@ mod tests {
         assert_eq!(token.tag.len(), TAG_LEN);
     }
 
-    #[test]
-    fn test_compute_expiry() {
-        let time_now = get_time_s(ClockType::Monotonic);
-        let expiry = TokenAuthority::compute_expiry(1);
-        assert_eq!(expiry - time_now, 1);
-
-        let time_now = get_time_s(ClockType::Monotonic);
-        let expiry = TokenAuthority::compute_expiry(0);
-        assert_eq!(expiry, time_now);
-    }
-
```

Comment:
I think we should replace this test instead of deleting it.

Reply:
Done.